test/TEST-16-UDEV-WANTS/test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+set -e
+TEST_DESCRIPTION="UDEV SYSTEMD_WANTS property"
```

Comment:
Some other PR is also adding TEST-16-* I think. So if you were respinning this, it'd be good to change to 17.